src/xdocs/google_style.xml

```diff
@@ -798,6 +799,11 @@
                   <a href="https://github.com/checkstyle/checkstyle/issues/14938">
                     #14938</a>.
                   <br />
+                  <br />
+                  To suppress "LineLength" violations for long identifiers,
+                  use following annotation:
+                  <code>@SuppressWarnings({"LineLength"})</code>
```

Comment:
Please postpone this update until we agree in that issue.

For now Please update table with comment: `long indentures will be handled at https://github.com/checkstyle/checkstyle/issues/15233`

Reply:
done